### DISCUSSION TITLE
Error while using flowbite with react

### DISCUSSION THREAD

**Discussion by Balikuddembe** [themesberg/flowbite#724] [Q&A]
Hello folks!

I am building a project with react,tailwind and flowbite. Flow bite is not automatically applying event listener and i have an error below in the console

```WARNING in ./node_modules/flowbite/lib/esm/index.js
Module Warning (from ./node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'D:\Aviation-IT-Systems\ANS-SMS-Java-React\node_modules\flowbite\src\index.ts' file: Error: ENOENT: no such file or directory, open 'D:\Aviation-IT-Systems\ANS-SMS-Java-React\node_modules\flowbite\src\index.ts'```

HoW CAN I GO ABOUT THE ERROR?

**Comment by zoltanszogyenyi** [comment]
Hey @Balikuddembe,

Thanks for using Flowbite!

Are you using the official Flowbite React project from here?

https://flowbite-react.com/

Cheers,
Zoltan

**Reply by Balikuddembe** [reply]
Hey @zoltanszogyenyi ,
I was using https://flowbite.com

Let me try to use the official flowbite react project.

regards,
Emmanuel
